Build error: "apksigner: not found"

After fixing #54 manually, I get another build error:

```
apksigner sign --in "_build/juloo.keyboard2.debug.unsigned-apk" --out "_build/juloo.keyboard2.debug.apk" \
        --ks _build/debug.keystore --ks-key-alias debug --ks-pass "pass:debug0"
/bin/sh: 1: apksigner: not found
make: *** [Makefile:59: _build/juloo.keyboard2.debug.apk] Error 127
rm _build/juloo.keyboard2.debug.unaligned-apk
```

For me, this simple fix works:

```diff
-    apksigner sign --in "$<" --out "$@" \
+    $(ANDROID_BUILD_TOOLS)/apksigner sign --in "$<" --out "$@" \
```

Is `apksigner` expected to be on the PATH?

--- a/Makefile
+++ b/Makefile
@@ -55,7 +55,7 @@ $(DEBUG_KEYSTORE):
 		-keyalg rsa -storepass $(DEBUG_PASSWD) -validity 10000
 
 _build/%.debug.apk: _build/%.debug.unsigned-apk $(DEBUG_KEYSTORE)
-	apksigner sign --in "$<" --out "$@" \
+	$(ANDROID_BUILD_TOOLS)/apksigner sign --in "$<" --out "$@" \
 		--ks $(DEBUG_KEYSTORE) --ks-key-alias debug --ks-pass "pass:$(DEBUG_PASSWD)"
 
 # Debug apk

